osi_hostvehicledata.proto

```diff
@@ -106,17 +106,17 @@ message HostVehicleDataClass
         //
         optional Pedalry pedalry = 1;
 
-        // Rounds per minute of the engine.
+        // Rounds per minute of the engine. RPM can be from E-Motor/ Engine.
```

Comment:
Better now! Thx :)

Reply:
since you brought the E-Motor stuff up; how to deal w/ vehicles, which have more than one of those (e.g. Tesla S 4x4)?

Reply:
Looks to me like we will have to do a repeated submessage "Engine". Good point, i will pay attention to that at the next steps ::)

Reply:
@ThomasNaderBMW in yesterday's TrafficParticipants discussion @pmai has also mentioned that we may need to consider how to set this signal in cases where more than one e-motor is available?
Idea for further discussion: It can be a consolidated signal going to wheels from all sources. Naming of the signal to be brainstormed in the follow-up meetings.

Reply:
Topic for next WP15-Meeting.